code/game/objects/structures/fireaxe.dm

```diff
@@ -65,7 +65,7 @@
 	else
 		return ..()
 
-/obj/structure/fireaxecabinet/play_attack_sound(damage_amount, damage_type = BRUTE, damage_flag = 0)
+/obj/structure/fireaxecabinet/play_attack_sound(damage_amount, damage_type = BRUTE, damage_flag = NONE)
```

Comment:
Aren't damage flags text defines ? Passing in 0 seems wrong here.

Reply:
Looks like they should be
Damage_flag is also not used anywhere in play_attack_sound 🙄